README.md

```diff
@@ -1,49 +1,83 @@
-# AWS <$module-name> Terraform module
+# AWS aws-backup Terraform module
```

Comment:
Maybe rename to `AWS Backup Terraform module`

Reply:
renamed